src/corelibs/U2View/src/ov_sequence/sequence_info/SequenceInfo.h

```diff
@@ -164,25 +164,25 @@ private slots:
     QSharedPointer<BaseTempCalc> temperatureCalculator;
 
     static const int COMMON_STATISTICS_TABLE_CELLSPACING;
```

Comment:
nit: all these private constants can be moved into cpp. It will reduce code size and re-compilation time caused by change in this private data (like in your example now)

Reply:
OK